leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/LeshanClusterServer.java

```diff
@@ -30,30 +30,32 @@
 import org.eclipse.leshan.server.client.ClientRegistry;
 import org.eclipse.leshan.server.model.LwM2mModelProvider;
 import org.eclipse.leshan.server.model.StandardModelProvider;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import redis.clients.jedis.Jedis;
 import redis.clients.jedis.JedisPool;
 import redis.clients.util.Pool;
 
+/**
+ * The main entry point for the Leshan LWM2M cluster server node.
+ * 
+ * Will start a LWM2M server which would be part of a large cluster group. This group of cluster node will share state
+ * and communicate using Redis. So they all need too be connected to the same redis server.
+ */
 public class LeshanClusterServer {
-    private static final Logger LOG = LoggerFactory.getLogger(LeshanClusterServer.class);
 
     private final static String USAGE = "java -jar leshan-server-cluster.jar [OPTION]";
-    private final static String FOOTER = "All options could be passed using environment variables.(using long option name in uppercase)";
+    private final static String FOOTER = "Question? Bug report? Eclipse Leshan : https://eclipse.org/leshan";
 
     public static void main(String[] args) {
         // Define options for command line tools
         Options options = new Options();
 
-        options.addOption("h", "help", false, "Display help information.");
-        options.addOption("n", "instanceID", true, "Set the unique identifier of this instance in the cluster.");
-        options.addOption("lh", "coaphost", true, "Set the local CoAP address.\n  Default: any local address.");
-        options.addOption("lp", "coapport", true, "Set the local CoAP port.\n  Default: 5683.");
-        options.addOption("slh", "coapshost", true, "Set the local secure CoAP address.\nDefault: any local address.");
-        options.addOption("slp", "coapsport", true, "Set the local secure CoAP port.\nDefault: 5684.");
-        options.addOption("wp", "webport", true, "Set the HTTP port for web server.\nDefault: 8080.");
+        options.addOption("h", "help", false, "Display this help message.");
+        options.addOption("n", "instanceID", true, "Sets the unique identifier of this instance in the cluster.");
+        options.addOption("l", "coapaddressport", true,
+                "Sets the local CoAP bind address and port in form [IP]:[PORT].\n  Default: 0.0.0.0:5683.");
+        options.addOption("sl", "coapsaddressport", true,
+                "Sets the local secure CoAP bind address and port, same format.\n  Default: 0.0.0.0:5684.");
         options.addOption("r", "redis", true,
```

Comment:
I think the number of option on the clluster node will grow so having 6 options for port/iface mapping was large.
and -slh -slp :grin: looks so simple. 
BTW I did it for reducing the number of port/address options (6) but then I removed the web port only config which is not only inconsistent (no iface config) but also unused. So we can go back to the previous solution if you remove the web port binding option.